### PR TITLE
New version: Google.PlatformTools version 35.0.2

### PR DESCRIPTION
**Pull request by ponces**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/190804)

### CODE REVIEW

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- RelativeFilePath: platform-tools\fastboot.exe
+  PortableCommandAlias: fastboot
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.google.com/android/repository/platform-tools_r35.0.2-win.zip
+  InstallerSha256: 2975A3EAC0B19182748D64195375AD056986561D994FFFBDC64332A516300BB9
+  ArchiveBinariesDependOnPath: true
+ManifestType: installer
+ManifestVersion: 1.9.0
```
Comment (stephengillie) [inline manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.installer.yaml:20]:
```suggestion
ManifestVersion: 1.7.0
```

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- debug
+- development
+- fastboot
+- flash
+- platform-tools
+Documentations:
+- DocumentLabel: Android Debug Bridge (adb)
+  DocumentUrl: https://developer.android.com/tools/adb
+ManifestType: defaultLocale
+ManifestVersion: 1.9.0
```
Comment (stephengillie) [inline manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml:32]:
```suggestion
ManifestVersion: 1.7.0
```

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: Google.PlatformTools
+PackageVersion: 35.0.2
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.9.0
```
Comment (stephengillie) [inline manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.yaml:8]:
```suggestion
ManifestVersion: 1.7.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @ponces,

I'm not entirely sure, but I believe that `1.7.0` is the highest manifest version currently supported.

**Comment by ponces** [comment]
> Hello @ponces,
> 
> The package manager bot determined that the ManifestVersion key does not match any version currently approved for release. Please verify the manifest file is compliant with the package manager [1.5 manifest specification](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0) or [1.6 manifest specification](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0).
> 
> For details on the specific error, see the details link below in the build pipeline.
> 
> Template: msftbot/validationError/manifest/version/error

> > The thing is the flag `ArchiveBinariesDependOnPath` was only introduced on `1.9.0`. AFAIK this is the highest manifest version, though it's not wide published yet. The fix was merged in [microsoft/winget-cli#4816](https://github.com/microsoft/winget-cli/pull/4816). I can locally use this fix while using `winget-cli v1.9.25180` and these new manifests.
> > 
> > This flag fixes the issues [microsoft/winget-cli#2711](https://github.com/microsoft/winget-cli/issues/2711) and [microsoft/winget-cli#2889](https://github.com/microsoft/winget-cli/issues/2889) for this package. Is there any plan to publish the new `1.9.0` manifest version? I can leave this PR as a draft until then.

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.installer.yaml (hunk 1 above)
Comment (ponces) [reply manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.installer.yaml:20]: Manifest 1.9.0 is [live](https://github.com/microsoft/winget-pkgs/pull/195988).

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml (hunk 2 above)
Comment (ponces) [reply manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml:32]: Manifest 1.9.0 is [live](https://github.com/microsoft/winget-pkgs/pull/195988).

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.yaml (hunk 3 above)
Comment (ponces) [reply manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.yaml:8]: Manifest 1.9.0 is [live](https://github.com/microsoft/winget-pkgs/pull/195988).

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+PackageIdentifier: Google.PlatformTools
+PackageVersion: 35.0.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: platform-tools\adb.exe
+  PortableCommandAlias: adb
+- RelativeFilePath: platform-tools\fastboot.exe
+  PortableCommandAlias: fastboot
+UpgradeBehavior: uninstallPrevious
```
Comment (aisgbnok) [inline manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.installer.yaml:13]:
```suggestion
UpgradeBehavior: uninstallPrevious
ReleaseDate: 2024-07-26
```

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Google.PlatformTools
+PackageVersion: 35.0.2
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://developer.android.com/
+PublisherSupportUrl: https://developer.android.com/tools#tools-platform
+PrivacyUrl: https://policies.google.com/privacy
+PackageName: Android SDK Platform-Tools
```
Comment (aisgbnok) [inline manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml:11]:
```suggestion
PackageName: Android SDK Platform Tools
```

Google uses "SDK Platform Tools" on their site. No dash.

File: manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  It includes tools that interface with the Android platform, primarily adb and fastboot.
+Tags:
+- adb
+- android
+- debug
+- development
+- fastboot
+- flash
+- platform-tools
+Documentations:
```
Comment (aisgbnok) [inline manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml:28]:
```suggestion
- platform-tools
ReleaseNotes: |-
  adb
  - Fix openscreen mDNS backend bug bringing down server on truncated query issue #294120933.
  - Make openscreen mDNS backend work on macOS.
  - Make openscreen mDNS backend default on all platforms.
  - Support to detect USB SuperSpeed+ (current and negotiated speeds) for diagnostic purposes.
  - Graceful shutdown: Release all USB interfaces on shutdown (all OSes).
ReleaseNotesUrl: https://developer.android.com/tools/releases/platform-tools#3502_july_2024
Documentations:
```

**Review by aisgbnok — CHANGES_REQUESTED** [review]
Just a few suggestions/fixes.

---

I verified this PR locally. If the validation pipeline is rerun this will pass.

**Comment by ponces** [comment]
> Just a few suggestions/fixes.
> 
> I verified this PR locally. If the validation pipeline is rerun this will pass.

Thanks

**Review by aisgbnok — APPROVED** [review]
Thank you.

<details>
  <summary></summary>

  ```PowerShell
--> Installing WinGet
--> Disabling safety warning when running installers
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Enabled admin setting 'LocalManifestFiles'.
Enabled admin setting 'LocalArchiveMalwareScanOverride'.

--> Installing the Manifest 35.0.2

Found Android SDK Platform-Tools [Google.PlatformTools] Version 35.0.2
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.google.com/android/repository/platform-tools_r35.0.2-win.zip
  ██████████████████████████████  6.39 MB / 6.39 MB
Successfully verified installer hash
Extracting archive...
Successfully extracted archive
Starting package install...
Path environment variable modified; restart your shell to use the new value.
Command line alias added: "adb"
Command line alias added: "fastboot"
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName                DisplayVersion Publisher  ProductCode                         Scope
-----------                -------------- ---------  -----------                         -----
Android SDK Platform-Tools 35.0.2         Google LLC Google.PlatformTools__DefaultSource User


PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> adb --version
Android Debug Bridge version 1.0.41
Version 35.0.2-12147458
Installed as C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\WinGet\Packages\Google.PlatformTools__DefaultSource\platform-tools\adb.exe
Running on Windows 10.0.26100
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> fastboot --version
fastboot version 35.0.2-12147458
Installed as C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\WinGet\Packages\Google.PlatformTools__DefaultSource\platform-tools\fastboot.exe
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs>
  ```
</details>

**Review by ponces — COMMENTED** [review]
Is everything ok?

**Comment by stephengillie** [comment]
Since logs are not currently emitted, my automation can't filter them to find the error. And there is too much PR volume for me to manually verify all errors. So I am only verifying those with discussion. 
- https://github.com/microsoft/winget-pkgs/issues/194328

**Review by stephengillie — APPROVED** [review]

**Pull request merged**